py3dtilers/TilesetReader/parsed_b3dm.py

```diff
@@ -0,0 +1,130 @@
+import os
+
```

Comment:
Same comment as for tileset_creation.py (allthough these are already classes).
The only exposed method should here be ParsedB3dm::parse_tile() that could be renamed to convert().